package.json

```diff
@@ -8,7 +8,7 @@
     },
     "scripts": {
         "test": "echo \"Error: no test specified\" && exit 1",
-        "start": "node openapi-gui.js"
+        "start": "node --harmony openapi-gui.js"
```

Comment:
Is this necessary on supported (>=4.x) versions of Node?

Reply:
I don't know.  Refer to https://nodejs.org/en/docs/es6/.  However, I believe this switch is unnecessary given the way widdershins and shins APIs are written.

Give me a sec and I'll strip these issues out and run a quick test.

Reply:
Fixed.